Quality check: Metadata updates in AAC Content Pattern Library (#42)
MicrosoftDocs/content-templates: templates/org/aac/aac-example-workload-content.md

> <!--Add the aac-browse-header.yml here-->

Reviewer: ```suggestion
<!--Add the aac-browse-header.yml here.-->
```

> <!--Add the aac-browse-header.yml here.-->

Reviewer: Do you think readers will know where the aac-browse-header.yml file is located? Can we provide a clearer instruction?

-->

## Architecture
<!--
- Visio or PowerPoint files aren't allowed in the GitHub repo. Send the file or provide a link so that the file can be uploaded to our limited-access CDN server.
- Ensure that alt text is descriptive, between 40 and 150 characters, and ends with a period.
- Update the following image syntax for the architecture diagram, and the Visio file or PowerPoint file text.
-->

![Diagram that shows the <solution name> architecture.](./images/<file-name>.png)

*Download a [Visio file](https://arch-center.azureedge.net/[file-name].vsdx) of this architecture.*

### Dataflow
<!--
- Title this section **Workflow** if data isn't in the scenario.
- Include a numbered list that describes the dataflow or workflow of each step in the solution. Start from the user or external data source, and then follow the flow through the rest of the solution. The following section uses the previous diagram as an example.
-->

> The following dataflow corresponds to the previous diagram:

Reviewer: I think this line needs

<!--
The Example workload template can include a metadata block within the article or uses the Azure Architecture Center browser header for its metadata.
  - Most example workload articles use the standard Learn format, which is located in a single file in a YML block at the top of the article. Use the standard Markdown format if the content of the article is conceptual, process oriented, or implementation level like a how-to guide. For more information, see [Metadata in the standard Markdown format](help/patterns/level4/aac-example-workload-instructions?branch=main#metadata-in-the-standard-markdown-format).

>   - An Azure Architecture Center browser header supports the browser by containing the metadata, the link to the thumbnail, and hyperlinks of one or more content markdown files. These architecture guides appear in the AAC browse experience for customers and support workloads that focus on architectural diagrams. If you use the browse format, you will need to create a separate YML file for the article metadata. For more information, see [Template for the Azure Architecture Center browser header](/help/patterns/level4/aac-browser-header-template)

Reviewer: ```suggestion
  - An Azure Architecture Center browser header supports the browser by containing the metadata, the link to the thumbnail, and hyperlinks of one or more content Markdown files. These architecture guides appear in the AAC browse experience for customers and support workloads that focus on architectural diagrams. If you use the browse format, you will need to create a separate YML file for the article metadata. For more information, see [Template for the Azure Architecture Center browser header](/help/patterns/level4/aac-browser-header-template).
```

Security provides assurances against deliberate attacks and the abuse of your valuable data and systems. For more information, see [Design review checklist for Security](/azure/well-architected/security/checklist).

<!--
- Include identity and data sovereignty considerations in this section.
- Describe any security considerations that aren't typical.
- Because security is important to our business, be sure to include your Azure security baseline assessment recommendations in this section. See https://aka.ms/AzureSecurityBaselines.
-->

### Cost optimization
<!--
- REQUIRED: This section is required. Cost is of the utmost importance to our customers.
- REQUIRED STATEMENT: Include the following statement to introduce the section:
-->

Cost optimization is about looking at ways to reduce unnecessary expenses and improve operational efficiencies. For more information, see [Design review checklist for Cost Optimization](/azure/well-architected/cost-optimization/checklist).

<!--
- Answer the following questions:
    - How much will this cost to run? Try to answer without using dollar amounts.
    - Are there ways I could save cost?

>     - If it scales linearly, then should we break it down by cost or unit. If it doesn't scale linearly, why?

Reviewer: ```suggestion
    - If it scales linearly, then should we break it down by cost or unit? If it doesn't scale linearly, why?
```

> Cost Optimization is about looking at ways to reduce unnecessary expenses and improve operational efficiencies. For more information, see [Design review checklist for Cost Optimization](/azure/well-architected/cost-optimization/checklist).

Reviewer: ```suggestion
Cost Optimization is about considering ways to reduce unnecessary expenses and improve operational efficiencies. For more information, see [Design review checklist for Cost Optimization](/azure/well-architected/cost-optimization/checklist).
```
This phrasing will be easier to translate.

    - What components make up the cost?
    - How does scale affect the cost?
- Link to the pricing calculator (<https://azure.microsoft.com/pricing/calculator>). Include all the components in the architecture, even if they have a $0 or $1 usage.
- Include small, medium, or large configurations if it makes sense to. Describe what needs to be changed when you move to larger sizes.
-->

### Operational Excellence
<!--REQUIRED STATEMENT: If you use this section, include the following statement:-->

> Operational Excellence covers the operations processes that deploy an application and keep it running in production. For more information, see [Design review checklist for Operational Excellence](/azure/well-architected/operational-excellence/checklist).

Reviewer: ```suggestion
Operational Excellence addresses the operations processes that deploy an application and keep it running in production. For more information, see [Design review checklist for Operational Excellence](/azure/well-architected/operational-excellence/checklist).
```

<!--
- Include scalability considerations in this section.
- Answer the following questions:
  - Are there any key performance considerations beyond the typical?`
  - Are there any size considerations around this solution specifically?
  - What scale does this work at?  
  - At what point do things break or not make sense for this architecture?
-->

## Deploy this scenario
<!--
- This section is optional but greatly encouraged.
- Answer the following questions:
  - Is there an example deployment that can show me this in action?
  - What would I need to change to run this in production?
-->

## Contributors
<!--This section is expected but optional if all the contributors prefer not to include it.

1. Start with the explanation text in italics that's the same for every article. This makes it clear that Microsoft takes responsibility for the article and not a single contributor.

1. Include the list of "Principal authors" and the list of "Additional contributors," if there are any. Format in plain text, not italics or bold.

1. Link each contributor's name to the person's LinkedIn profile. 

> 1. Insert a pipe symbol ("|") with spaces after the name, and then enter the person's title. To minimize edits or updates, don't include the person's company, MVP status, or links to additional profiles. You can link profiles from the person's LinkedIn page. In the future, we aim to automate this process on the platform.

Reviewer: ```suggestion
1. Insert a pipe symbol ("|") with spaces after the name, and then enter the person's title. To minimize edits or updates, don't include the person's company, MVP status, or links to additional profiles. 
```
